conda-recipe/meta.yaml

```diff
@@ -17,12 +17,14 @@ requirements:
     - numpy
     - six
     - matplotlib
+    - enaml
```

Comment:
What's the difference between the build and the run?  If we have a package in the build reqs do we also need it in the runs reqs?